Localization/de-DE/start-folding.yaml

```
@@ -0,0 +1,47 @@
+---
+url: "https://foldingathome.org/"
+language: "de-de"
```

```suggestion
language: "de-DE"
```
Has to be the same as in the folder name, I guess.

---

Ok